web/frontend/src/components/remote-control-cards.vue

```diff
@@ -644,7 +646,7 @@ const doConnect = async (authEntity: string, creds: Credentials, onError?: (reas
   }
 
   console.debug('connected');
-  document.querySelector('#pre-app')!.classList.add('hidden');
+  showAuth = false;
```

Comment:
I also replaced as many `.querySelector()` calls as possible to reduce the risk of querying something that could be unmounted.